.github/workflows/documentation.yml

```
@@ -0,0 +1,29 @@
+
+name: Documentation
+on:
+  push:
+    branches:
+    - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
```

```suggestion
        fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
```

Love that there's a comment! Don't understand what it's about.